spec/src/main/asciidoc/core/packagingdeployment.asciidoc

```diff
@@ -2,94 +2,86 @@
 
 == Packaging and deployment
 
-When an application is started, the container must perform _bean discovery_, detect definition errors and deployment problems and raise events that allow portable extensions to integrate with the deployment lifecycle.
+In {cdi_lite}, the term _deployment time_ means before the application is started, such as during application compilation, or during application startup at latest.
```

Comment:
This sounds a little weird as an opening sentence of a paragraph. Maybe swap places with the following one?

Reply:
Makes sense, will do.